eng/restore/harvestPackages.targets

```diff
@@ -23,6 +23,7 @@
 
     <!-- Allow to override package download and versions in case there is already a PackageDownload set -->
     <ItemGroup>
+      <PackageDownload Include="System.Text.Encodings.Web" Version="4.5.1" />
```

Comment:
Why was this necessary?

Reply:
This was necessary to avoid harvesting from 4.7.1 package. I think we can delete this now cc @safern